transform/tests/services/jms/conftest.py

```diff
@@ -0,0 +1,68 @@
+"""Fixtures used across most jms tests"""
+
+import uuid
+from typing import Any, Coroutine, Generator
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+
+from app.services.jms.connector import JMSMessageHandler, connect
+from app.settings import settings
+
+
+@pytest.fixture
+def mock_stomp_connection() -> Generator[MagicMock, None, None]:
+    """Provide a mock for stomp.Connection"""
+    with patch("stomp.Connection") as mock:
+        yield mock()
+
+
+@pytest.fixture
+def subscription_id() -> str:
+    """Generate a unique subscription ID."""
+    return f"{settings.STOMP_CLIENT_NAME}-{uuid.uuid4()}"
+
+
+@pytest.fixture
+def jms_handler(mock_stomp_connection: MagicMock) -> JMSMessageHandler:
```

Comment:
`def jms_handler(mock_stomp_connection: MagicMock) -> JMSMessageHandler:` -> 
`def jms_handler(mock_stomp_connection: MagicMock, subscription_id: str) -> JMSMessageHandler:`
      `    subscription_id=subscription_id`

Reply:
Good catch, I forgot about it even after all that struggle with tests 👀